### ISSUE TITLE
Copy the term in the string with right click on original

### ISSUE THREAD

**Issue by Mte90** [GlotPress/GlotPress-WP#750]
I do that in https://github.com/Mte90/GlotDict/blob/master/js/glotdict.js#L41

As example: in a string there is a term with a glossary translation, so can be boring to read the tooltip and write, instead with a right click on that term in the textare o f the translated string will put the glossary term in the locale in using.

**Comment by toolstack** [comment]
So the term gets appended to the end of the translation string? 

Would #754 be a better way to do this?

**Comment by toolstack** [comment]
I didn't like the way it just added it to the end of the string so I've created a PR that places the term at the cursor position in the text.

This allows you to highlight the term in the translation and then just right click the glossary term to replace it with the translation.

**Comment by Mte90** [comment]
Ok is good for me in any case, the important is that copy feature :-)

**Comment by ocean90** [comment]
What was the reason for choosing a right click? Why not a simple (left) click? On mobile there is no right click and I don't like disabling the context menu for that.

**Comment by toolstack** [comment]
There's no hover on mobile either 😉 (or its often horrible hard to trigger).

I wonder if left click might get triggered by accident if people tend to click on the term to view it?

**Comment by ocean90** [comment]
It's actually more complicated because a term can have more than one translations:

<img width="404" alt="bildschirmfoto 2017-07-02 um 14 37 57" src="https://user-images.githubusercontent.com/617637/27769876-3bc02f26-5f34-11e7-8315-9a75184d53bc.png">

So a click on the translation in the tooltip is probably the way to go.

**Comment by toolstack** [comment]
True, the current code copies the first one, I'm thinking the tooltip is probably not the best solution to displaying the glossary terms.

We could add an additional popup when there is more than one term so the user could select the one they want.

**Comment by Mte90** [comment]
The right click idea was because is very fast (over on the term) and also because GlotDict add a consistency link to the glossary terms.
With multiple terms for the problem but we can do:

- click on term, if is one copy without problems, with two open a dialog with hotkey to chose the term. In case if the term are more than one and are the same the behaviour can be the first one

**Comment by ocean90** [comment]
We could just add an `Insert`/`Use` button/link next to the term. Clicking on it will insert the term into the editor.

**Comment by Mte90** [comment]
Ok so you suggest to add a button near the term and avoid the right click.
Seems not a bad idea, with copy from original we can avoid people that doing manually a copy and paste.
I will work on that.

**Comment by Mte90** [comment]
The problem mainly is about a term with multiple translation so an idea can be a view with the list of terms and on click insert in the textarea.
